test/wallet_addEthereumChain.test.ts

```diff
@@ -26,5 +21,38 @@ describe("WalletRpcPlugin", () => {
         params: [{ chainId: "0x1388" }],
       });
     });
+
+    it("should return correct result", async () => {
+      const result = await web3.walletRpc.addEthereumChain({ chainId: 5000 });
+
+      expect(result).toBeUndefined();
+    });
```

Comment:
What is the purpose of this test?

Reply:
I don't know yet how to test actual wallet responses as each wallet can return whatever they want. I'm thinking to add some response parsers to always return the same type, despite wallet response, but I need to test it on real examples first (with real browser extension wallets). This test is not testing much yet, I agree.